src/libData/BlockData/Block/TxBlock.cpp

```diff
@@ -116,7 +116,7 @@ int TxBlock::Deserialize(const vector<unsigned char>& src, unsigned int offset)
                 = GetNumber<uint32_t>(src, curOffset, sizeof(uint32_t));
             curOffset += sizeof(uint32_t);
 
-            m_shardIDs.emplace_back(shardID);
+            m_shardIDs.push_back(shardID);
```

Comment:
Any reason why this can't be emplace_back?

Reply:
Please refer to this article: https://abseil.io/tips/112
There is always trade-off between performance & maintenance.
That is, for native data type, it doesn't matter to use push_back/emplace_back and we prefer to use push_back for good maintenance.
For user-defined data type, of course we prefer emplace_back for better performance.